### PR TITLE
Add changelog entry for experimental extensions

### PR DESCRIPTION
**Pull request by i14h**
Add changelog entry for experimental extensions

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 - Add warning when a developer is using yarn@2 PnP (#2198).
 - Fixes incorrect URLs reported inside emulated HTTPS functions (#1862).
 - Updates underlying timeout handler when proxying through the Hosting emulator.
+- Add support for deploying [experimental extensions](https://github.com/FirebaseExtended/experimental-extensions) (#2398)
```
Comment (jhuleatt) [inline CHANGELOG.md:4]:
```suggestion
- Add support for installing [experimental extensions](https://github.com/FirebaseExtended/experimental-extensions) (#2398) 
```

suggesting this to match the language we use in the [docs](https://firebase.google.com/docs/extensions/install-extensions)

Comment (rachelsaunders) [inline CHANGELOG.md:4]:
```suggestion
- Adds support for installing [experimental extensions](https://github.com/FirebaseExtended/experimental-extensions) (#2398) 
```

**Review by rachelsaunders — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jhuleatt) [inline CHANGELOG.md:4]:
```suggestion
- Add support for deploying [experimental extensions](https://github.com/FirebaseExtended/experimental-extensions) (#2830) 
```

I believe #2830 is correct. #2398 seems like a different PR

**Review by rachelsaunders — APPROVED** [review]

**Comment by i14h** [comment]
@jhuleatt @rachelsaunders fixed

**Review by jhuleatt — APPROVED** [review]

**Pull request merged**